src/stub/language.rs

```diff
@@ -21,7 +21,22 @@ pub(super) struct TypeTokens {
 pub(super) struct Language {
     pub variable_name_options: VariableNameOptions,
     pub source_file_ext: String,
+    /// NOTE: These comments are for a future PR
+    ///
+    /// Generic, used for either type keywords...:
+    /// - float    in:   float a_float;   (C)
+    /// - i32      in:   let an_int: i32; (Rust)
+    /// ... or parsing functions:
+    /// - int      in:   x = int(input()) i      (Python)
+    /// - StrToInt in:   x = StrToInt(Inputs[0]) (Pascal)
     pub type_tokens: TypeTokens,
+    /// But sometimes you need two tokens per type for a language.
+    /// - Int32 and StrToInt for Pascal.
+    pub type_parsers: Option<TypeTokens>,
+    /// And parsing symbols %d %f %ld/%lld.
+    /// Some languages share them for reading/writing while others
+    /// like Pascal / OCaml / F# use them only for printing (in CG)
+    pub type_extra_tokens: Option<TypeTokens>,
```

Comment:
I think these are more fitting as regular comments than doc comments (which show up in the documentation and on hover).

```suggestion
    // NOTE: These comments are for a future PR
    //
    // Generic, used for either type keywords...:
    // - float    in:   float a_float;   (C)
    // - i32      in:   let an_int: i32; (Rust)
    // ... or parsing functions:
    // - int      in:   x = int(input()) i      (Python)
    // - StrToInt in:   x = StrToInt(Inputs[0]) (Pascal)
    pub type_tokens: TypeTokens,
    // But sometimes you need two tokens per type for a language.
    // - Int32 and StrToInt for Pascal.
    pub type_parsers: Option<TypeTokens>,
    // And parsing symbols %d %f %ld/%lld.
    // Some languages share them for reading/writing while others
    // like Pascal / OCaml / F# use them only for printing (in CG)
    pub type_extra_tokens: Option<TypeTokens>,
```